src/VisualStudio/IntegrationTest/IntegrationTests/AbstractEditorTest.cs

```diff
@@ -36,14 +36,22 @@ protected AbstractEditorTest(VisualStudioInstanceFactory instanceFactory, string
         {
         }
 
-        protected AbstractEditorTest(VisualStudioInstanceFactory instanceFactory, string solutionName, string projectTemplate, bool clearEditor = true)
+        protected AbstractEditorTest(
+            VisualStudioInstanceFactory instanceFactory,
+            string solutionName,
+            string projectTemplate,
+            bool clearEditor = true,
+            bool setUseSuggestionMode = true)
```

Comment:
This name seems very odd: "clearSuggestionMode" seems a better name and much more inline with the existing pattern. I would have assumed "setUseSuggestionMode: true" means "set suggestion mode on".